### DISCUSSION TITLE
3.9.2.23版本V9怎么hook消息 我按照README.md 的描述实现不了

### DISCUSSION THREAD

**Discussion by CSL19980820** [ttttupup/wxhelper#200] [Q&A]
![image](https://github.com/ttttupup/wxhelper/assets/43056223/010f9438-00ff-4aee-a858-77e5e2023436)
是缺少或者多了什么参数吗

**Comment by ttttupup** [comment]
少了 url ，timeout.或者enableHttp设置0

**Reply by CSL19980820** [reply]
好的 谢谢
